lib/msal-browser/src/operatingcontext/BaseOperatingContext.ts

```diff
@@ -24,7 +24,7 @@ import {
  * For example: Some operating contexts will pre-cache tokens impacting performance telemetry
  */
 export abstract class BaseOperatingContext {
-    protected logger: Logger;
+    public logger: Logger;
```

Comment:
Should not be made public, if absolutely necessary there is a `getLogger` function on this class

Reply:
Done!